root/programs/C#/Frameworks/Infrastructure/CustomControl/Control/WebCustomTextBox.cs

```diff
@@ -326,12 +326,12 @@ public bool Validate(out string[] result)
             if (this.CheckType != null)
             {
                 // 必須入力チェック
-                if (this.CheckType.IsIndispensabile)
+                if (this.CheckType.IsIndispensable)
                 {
                     if ((text == ""))
                     {
                         hasError = true;
-                        lstRet.Add(CmnCheckFunction.IsIndispensabileCheckErrorMessage);
+                        lstRet.Add(CmnCheckFunction.IsIndispensableCheckErrorMessage);
```

Comment:
@daisukenishino 

Should be the type of "result" parameter "string[]"?
The following code does not use "lstRet".

```c#
public bool Validate(out List<string> result)
{
    // フラグ
    bool hasError = false;
    // 結果
    result = new List<string>();

    // omit

    hasError = true;
    result.Add(CmnCheckFunction.xxx);

    // omit

    return !hasError;
}
```